docs/Milestones.md

```diff
@@ -0,0 +1,12 @@
+# Milestones
+> Hitos
+
+## [[M0] Fichero con el modelo del problema](https://github.com/Carlos-SE/App-Chores/milestone/1)
+Los usuarios necesitan canciones en base a sus gustos musicales. Es por esto que se necesita desarrollar una estructura que modelice las canciones. Almacenando tanto los datos de la canción, como el archivo de la canción en cuestión.
+
+## [[M1] Biblioteca para extracción de atributos de canciones](https://github.com/Carlos-SE/App-Chores/milestone/3)
```

Comment:
Ya te dije ayer que este orden es incorrecto. Además, aquí hablas de "biblioteca" y a continuación de "implementación". No busquéis palabras mágicas que hay que poner para resolver el objetivo. Tratad de establecer un orden lógico en las fases de desarrollo de un producto. ¿Hace falta una "biblioteca" de extracción de "atributos" para la lógica de negocio?

Reply:
Se ha cambiado la biblioteca de extracción a una con la que trabajar con la estructura de datos que se creará para representar las canciones. Con este cambio, sí, es necesario que la biblioteca exista antes de la lógica de negocio ya que sino no se puede desarrollar dicha lógica de negocio.